### ISSUE TITLE
entries for sub-species are pooled together in the same species [bug]

### ISSUE THREAD

**Issue by jeffreyhanson** [bird-team/brisbane-birds-prototype#6]

**Comment by dbl3raf** [comment]
this is OK - subspecies reporting is too occasional in eBird to be of much use. There are possibly one or two cases where we might want to split things up by subspecies, but these can be identified separately

**Comment by jeffreyhanson** [comment]
At the moment the book just compares the species in the ebird data file with the species in the taxonomic datafile and generates species found in both.

I wonder if we should include another table specifying the species names that we want to include in the book?

So, only species that appear in the table, ebird and the taxonomy data would be in the book. This would let us explicitly control which sub-species we want in the book.

**Comment by dbl3raf** [comment]
good idea - I'll build the list. I've assigned this comment to me - hopefully that means it will somehow remind me that I need to do it
